Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,46 @@
+6. Some form of battery monitoring capabilities need to be included or applicable as per shall statements.
+7. The manufacturer needs to be a reputable and safe choice.  A stable manufacturer is assumed to remain in business for years to come and continue producing the chosen board is a necessity.  This project aims for replicability, so a board that remains relevant and available is a key part.  Public and customer reviews will be used to gauge reputability.
+8. The board will need to have coding language support for C and/or Python.  At least one must be supported. These languages are chosen due to popularity, online support, and performance.  This helps create a wider web of coverage for end-users being at least relatively experienced with coding and engineering concepts.
+
+## BUILDABLE SCHEMATIC OF SUBSYSTEM
+Shown is the pseudo-code for the operating code.  The final form of the code may have parts added and additional functionalities included, but this is the general idea of how the code should flow.  Proceeding from each section is assuming that there are no errors detected indicating a “Success”.  If an error arises, a default handler will need to be called to assess the situation.  It should also be noted that the loop is intended to be theoretically infinite, however, there are explicit and implicit breaks at the battery running critically low (explicit) and the shutoff button being pushed (implicit).  “Timers” refer to either physical hardware or coded timers that go off to alert the system that the sensor(s) sampling is ready.
+
+![pseudo code 2](https://user-images.githubusercontent.com/110966922/202583040-c28e27fe-c175-4287-a8cb-cefafbfe327b.png)
+
+Shown is a schematic of the Arduino Mega 2560 Rev 3.
+![CAPSTONE 1 Microcontroller Schematic](https://user-images.githubusercontent.com/118767661/204694703-fcaa5b67-bf5d-469f-a134-dae06493fe5f.png)
+
+## ANALYSIS OF CHOSEN COMPONENT(S)
+The team has chosen to look at the Arduino Mega 2560 Rev 3 and Arduino USB Shield Host as the solutions.  These components, in conjunction, will satisfy the constraints and jobs of this subsystem.  Following are the corresponding fulfillment of the constraints:
+
+1. The Arduino Mega 2560 Rev 3 falls within the preferred range of $20-$60 at a current price of $48.40 (as of 11-8-22).  The USB Host Shield component has a price of $20.99 as of the same date.   
+2. 4 UART serial hardware ports are present on the board.  SPI is supported as well with included libraries.  As an alternative later, Two-Wire Interface (TWI) is also supported.  These enable the Mega to issue instructions and exchange data between other components and subsystems (Communication).
+3. The USB Host Shield enables the use of USB devices such as a Flash drive for external storage and communication. The Arduino Mega does not support USB as a standalone component.
```

Comment:
How does that affect the usage of pins and the usb port for programming? It seems like this device should be easily programmed. So, dismantling to unplug a usb shield would seem a poor design. Also, the usb host shield was not a part of the design. So, the schematic is incomplete.